src/styles.css

```diff
@@ -0,0 +1,248 @@
+*,
+*::after,
+*::before {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+}
+
+:root {
+    overflow-x: hidden;
+}
+
+.pageBackground {
+    position: absolute;
+    top: 0;
+    width: 100vw;
+    height: 100vh;
+    transition: 300ms 100ms linear;
+    z-index: -10;
+}
+
+.pageBackground--scale {
+    background: black;
+    opacity: 0.7;
+    z-index: 20;
+}
+
+.screen-readers {
+    position: absolute;
+    right: 2400px;
+}
```

Comment:
opowiesz coś o tym? nie pisałem dla screen readerów więc może coś się dowiem ciekawego, ale na 1 rzut oka dziwnie mi to wygląda.